src/LuaTransformer.ts

```diff
@@ -5162,38 +5262,6 @@ export class LuaTransformer {
         }
     }
 
-    protected getAbsoluteImportPath(relativePath: string): string {
```

Comment:
Any reason you moved these? I think they were located correctly in this file.

Reply:
I mainly moved them to tsHelper as it was logic that _transpileAndExecuteProjectReturningMainExport_ needed.

The methods were not available to convert TypeScript path strings to Lua import and export path strings without constructing a LuaTransformer only to convert a string using those methods.